options.js

```diff
@@ -82,14 +96,24 @@ function getVals() {
 	localStorage['lineEnding'] = document.optform.lineEnding.value;
 	localStorage['copySeparator'] = document.optform.copySeparator.value;
 	localStorage['maxClipCopyEntries'] = document.optform.maxClipCopyEntries.value;
-
+	try {
+		if (!isFinite(parseInt(document.optform.popupDelay.value))) {

```

Comment:
why not save the parsed popupDelay value and then add it to localStorage.  That way you don't need to parseInt twice.